org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -19,8 +21,7 @@ class Translator {
         this._baseUrl = "https://api.wink.com";
         this._devicesPath = '/users/me/wink_devices';
         this._oAuthPath = '/oauth2/token';
-
-        this._name = "Wink Hub"; // TODO: Can be pulled from OpenT2T global constants. This information is not available, at least, on wink hub.
+        this._name = OpenT2TConstants.WinkHubName;
```

Comment:
I don't think we should put the provider names into the OpenT2T library as we'd have to add them each time a new provider was added (by anybody).  This should just be a string here, and get rid of the comment.

---
In reply to: [101874739](https://github.com/openT2T/translators/pull/158#discussion_r101874739) [](ancestors = 101874739)